pkgs/development/python-modules/allure-pytest/default.nix

```diff
@@ -18,13 +18,13 @@
 
 buildPythonPackage rec {
   pname = "allure-pytest";
-  version = "2.9.45";
+  version = "2.10.0";
 
   disabled = pythonOlder "3.4";
```

Comment:
Probably also only support for Python 3.7 and above as upstream doesn't check against Python releases which are EOL (e.g., https://github.com/allure-framework/allure-python/blob/master/.github/workflows/build.yaml#L101).